### ISSUE TITLE
Unable to process a list of coordinates using directions

### ISSUE THREAD

**Issue by blobbybadger** [GIScience/openrouteservice#786]
#### Here's what I did
I was able to install a local version of openrouteservice on my local machine using Docker. I had to tweak the app.config file to have the following:

```
ors.services.routing.profiles.default_params.maximum_distance = 200000
ors.services.routing.profiles.default_params.maximum_waypoints = 200
```

I am trying to process the following command:

```sh
curl -X POST 'http://localhost:8082/ors/v2/directions/driving-car/gpx' -H 'Content-Type: application/json; charset=utf-8' -H 'Accept: application/json, application/geo+json, application/gpx+xml, img/png; charset=utf-8' -d '{"coordinates":[[-87.9168846,42.9583366],[-87.9168725,42.9583412],[-87.9168599,42.9582899],[-87.9168918,42.9582942],[-87.916893,42.9583159],[-87.9168921,42.9583344],[-87.9169123,42.9583055],[-87.9168888,42.9583307],[-87.9168997,42.9583062],[-87.916886,42.9583352],[-87.9168782,42.9583073],[-87.9168922,42.9583151],[-87.9168905,42.958315],[-87.9169096,42.9583325],[-87.9169117,42.9583254],[-87.9169269,42.9582832],[-87.9163345,42.9584378],[-87.916627,42.9574551],[-87.91553,42.9586651],[-87.9475805,42.9595118],[-87.9475571,42.9594895],[-87.9486234,42.9595628],[-87.9482612,42.9795793],[-87.951156,42.99402],[-87.9510322,42.9938831],[-87.951977,42.9940065],[-87.9536587,42.9940534],[-87.955169,42.9939977],[-87.9579049,43.0030971],[-87.9577496,43.0333794],[-87.9772619,43.0367925],[-87.992265,43.0336758],[-87.9922682,43.033604],[-87.9922443,43.0327109],[-87.9933474,43.0332867],[-87.9924723,43.0338892],[-88.0000205,43.0358494],[-88.0366678,43.0350936],[-88.0473042,43.0241071],[-88.0529112,43.0232338],[-88.0517818,43.0237163],[-88.0472025,43.0237129],[-88.0465328,43.0242576],[-88.0471096,43.0237901],[-88.0533071,43.0247272],[-88.0533014,43.0247642],[-88.0535405,43.0239081],[-88.0469584,43.0156329],[-88.0556531,43.0083531],[-88.0555673,43.0075324],[-88.0557401,43.008418],[-88.0568062,43.0126481],[-88.0518452,43.0128916],[-88.0521107,43.0127273],[-88.0527378,43.0131722],[-88.0470176,43.0134494],[-88.0317175,43.0025737],[-87.9864016,43.0029008],[-87.9856561,43.0029048],[-87.9755603,43.0078832],[-87.9746562,43.0082468],[-87.9678956,43.0101215],[-87.9617795,43.010196],[-87.9607323,43.010231],[-87.9659848,43.0029334],[-87.9666883,43.0029545],[-87.9666883,43.0029545],[-87.967987,43.0029532],[-87.9956577,43.0028567],[-88.0268902,43.0026913],[-88.0523474,43.0027095],[-88.054555,43.004406],[-88.0546366,43.0052889],[-88.0319,43.0025914],[-88.0079276,43.0115608],[-87.9782634,43.0191452],[-87.9539831,43.0217911],[-87.9196989,43.0231886],[-87.9184276,43.0232477],[-87.9183973,43.0232854],[-87.9173488,43.0232475],[-87.9121198,43.0262895],[-87.9118902,43.0261065],[-87.9125916,43.0267069],[-87.9094621,43.0320941],[-87.9075285,43.0288772],[-87.9052248,43.0251038],[-87.9062828,43.0251932],[-87.9062244,43.0251756],[-87.9051873,43.0251311],[-87.9053374,43.0263937],[-87.9031771,43.0365599],[-87.9031306,43.0365606],[-87.9042894,43.0364767],[-87.8918648,43.0062183],[-87.8846378,42.9592299],[-87.9163934,42.9586454],[-87.9161331,42.9562396],[-87.9155129,42.9569091],[-87.9154828,42.9584507],[-87.9155177,42.9592366],[-87.9299871,42.9593697],[-87.9154267,42.97543],[-87.928499,43.0360168],[-87.9343348,43.0348143],[-87.9343475,43.0348728],[-87.9353638,43.0347717],[-87.9264918,43.0352277],[-87.9379952,43.0599605],[-87.958543,43.0761479],[-87.983033,43.0953096],[-88.0058333,43.1256024],[-88.0052597,43.1484166],[-88.0157592,43.1496855],[-88.0157251,43.1496387],[-88.0163915,43.1489665],[-88.0057463,43.1351389],[-87.9987111,43.1321613],[-87.9985976,43.1331105],[-88.0039219,43.1342334],[-88.0062576,43.156088],[-88.0056933,43.1537617],[-88.0055853,43.1543852],[-88.0122886,43.1568837],[-88.0125847,43.1569529],[-88.012591,43.1570219],[-88.0115547,43.1568351],[-87.9827013,43.1484302],[-87.9538692,43.1371659],[-87.9454563,43.1021863],[-87.9308695,43.087122],[-87.920871,43.0698662],[-87.9251849,43.0411356],[-87.9162395,43.0100252],[-87.9137087,42.9823403],[-87.9288849,42.9737793],[-87.9281101,42.9591459],[-87.9281184,42.9591356],[-87.9266011,42.9592854],[-87.9167001,42.9582797],[-87.9169082,42.9583198],[-87.9169085,42.9583411]]}
```

#### Here's what I got

But I am getting the following error:

```
{"error":{"code":2004,"message":"Request parameters exceed the server configuration limits. The approximated route distance must not be greater than 100000.0 meters."},"info":{"engine":{"version":"6.2.0","build_date":"2020-07-31T19:33:10Z"},"timestamp":1596224251626}}
```

https://ask.openrouteservice.org/t/unable-to-process-a-list-of-coordinates-using-directions/1968

#### Here's what I was expecting
I understand what it is saying but what distance do I need to tweak? I tried changing all variables with the word distance in them to 200000 but then I get a Java error about being out of boundaries. The coordinates I listed above work properly when I use a gpx file 

```xml
<?xml version="1.0" encoding="utf-8" standalone="yes"?>
<gpx version="1.1" creator="GPS Visualizer https://www.gpsvisualizer.com/" xmlns="http://www.topografix.com/GPX/1/1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.topografix.com/GPX/1/1 http://www.topografix.com/GPX/1/1/gpx.xsd">
<trk>
  <name>BreadcrumbData-2017-08_28306_20170831</name>
  <trkseg>
    <trkpt lat="42.9583366" lon="-87.9168846">
      <time>2017-08-31T00:03:04Z</time>
    </trkpt>
    <trkpt lat="42.9583412" lon="-87.9168725">
      <time>2017-08-31T01:03:04Z</time>
    </trkpt>
    <trkpt lat="42.9582899" lon="-87.9168599">
      <time>2017-08-31T02:03:06Z</time>
    </trkpt>
    <trkpt lat="42.9582942" lon="-87.9168918">
      <time>2017-08-31T03:03:04Z</time>
    </trkpt>
    <trkpt lat="42.9583159" lon="-87.916893">
      <time>2017-08-31T03:50:46Z</time>
    </trkpt>
    <trkpt lat="42.9583344" lon="-87.9168921">
      <time>2017-08-31T04:03:04Z</time>
    </trkpt>
    <trkpt lat="42.9583055" lon="-87.9169123">
      <time>2017-08-31T05:03:04Z</time>
    </trkpt>
    <trkpt lat="42.9583307" lon="-87.9168888">
      <time>2017-08-31T06:03:04Z</time>
    </trkpt>
    <trkpt lat="42.9583062" lon="-87.9168997">
      <time>2017-08-31T07:03:06Z</time>
    </trkpt>
    <trkpt lat="42.9583352" lon="-87.916886">
      <time>2017-08-31T08:03:04Z</time>
    </trkpt>
    <trkpt lat="42.9583073" lon="-87.9168782">
      <time>2017-08-31T08:39:04Z</time>
    </trkpt>
    <trkpt lat="42.9583151" lon="-87.9168922">
      <time>2017-08-31T09:03:04Z</time>
    </trkpt>
    <trkpt lat="42.958315" lon="-87.9168905">
      <time>2017-08-31T10:03:04Z</time>
    </trkpt>
    <trkpt lat="42.9583325" lon="-87.9169096">
      <time>2017-08-31T11:03:04Z</time>
    </trkpt>
    <trkpt lat="42.9583254" lon="-87.9169117">
      <time>2017-08-31T12:03:04Z</time>
    </trkpt>
    <trkpt lat="42.9582832" lon="-87.9169269">
      <time>2017-08-31T12:49:02Z</time>
    </trkpt>
    <trkpt lat="42.9584378" lon="-87.9163345">
      <time>2017-08-31T12:53:02Z</time>
    </trkpt>
    <trkpt lat="42.9574551" lon="-87.916627">
      <time>2017-08-31T12:56:10Z</time>
    </trkpt>
    <trkpt lat="42.9586651" lon="-87.91553">
      <time>2017-08-31T12:57:10Z</time>
    </trkpt>
    <trkpt lat="42.9595118" lon="-87.9475805">
      <time>2017-08-31T13:02:16Z</time>
    </trkpt>
    <trkpt lat="42.9594895" lon="-87.9475571">
      <time>2017-08-31T13:03:04Z</time>
    </trkpt>
    <trkpt lat="42.9595628" lon="-87.9486234">
      <time>2017-08-31T13:04:10Z</time>
    </trkpt>
    <trkpt lat="42.9795793" lon="-87.9482612">
      <time>2017-08-31T13:08:00Z</time>
    </trkpt>
    <trkpt lat="42.99402" lon="-87.951156">
      <time>2017-08-31T13:13:00Z</time>
    </trkpt>
    <trkpt lat="42.9938831" lon="-87.9510322">
      <time>2017-08-31T13:13:26Z</time>
    </trkpt>
    <trkpt lat="42.9940065" lon="-87.951977">
      <time>2017-08-31T13:21:28Z</time>
    </trkpt>
    <trkpt lat="42.9940534" lon="-87.9536587">
      <time>2017-08-31T13:23:14Z</time>
    </trkpt>
    <trkpt lat="42.9939977" lon="-87.955169">
      <time>2017-08-31T13:35:22Z</time>
    </trkpt>
    <trkpt lat="43.0030971" lon="-87.9579049">
      <time>2017-08-31T13:38:00Z</time>
    </trkpt>
    <trkpt lat="43.0333794" lon="-87.9577496">
      <time>2017-08-31T13:43:00Z</time>
    </trkpt>
    <trkpt lat="43.0367925" lon="-87.9772619">
      <time>2017-08-31T13:48:00Z</time>
    </trkpt>
    <trkpt lat="43.0336758" lon="-87.992265">
      <time>2017-08-31T13:51:16Z</time>
    </trkpt>
    <trkpt lat="43.033604" lon="-87.9922682">
      <time>2017-08-31T14:03:04Z</time>
    </trkpt>
    <trkpt lat="43.0327109" lon="-87.9922443">
      <time>2017-08-31T14:03:56Z</time>
    </trkpt>
    <trkpt lat="43.0332867" lon="-87.9933474">
      <time>2017-08-31T14:05:40Z</time>
    </trkpt>
    <trkpt lat="43.0338892" lon="-87.9924723">
      <time>2017-08-31T14:10:46Z</time>
    </trkpt>
    <trkpt lat="43.0358494" lon="-88.0000205">
      <time>2017-08-31T14:13:00Z</time>
    </trkpt>
    <trkpt lat="43.0350936" lon="-88.0366678">
      <time>2017-08-31T14:18:00Z</time>
    </trkpt>
    <trkpt lat="43.0241071" lon="-88.0473042">
      <time>2017-08-31T14:23:00Z</time>
    </trkpt>
    <trkpt lat="43.0232338" lon="-88.0529112">
      <time>2017-08-31T14:25:08Z</time>
    </trkpt>
    <trkpt lat="43.0237163" lon="-88.0517818">
      <time>2017-08-31T14:32:00Z</time>
    </trkpt>
    <trkpt lat="43.0237129" lon="-88.0472025">
      <time>2017-08-31T14:33:00Z</time>
    </trkpt>
    <trkpt lat="43.0242576" lon="-88.0465328">
      <time>2017-08-31T14:34:02Z</time>
    </trkpt>
    <trkpt lat="43.0237901" lon="-88.0471096">
      <time>2017-08-31T14:58:34Z</time>
    </trkpt>
    <trkpt lat="43.0247272" lon="-88.0533071">
      <time>2017-08-31T14:59:52Z</time>
    </trkpt>
    <trkpt lat="43.0247642" lon="-88.0533014">
      <time>2017-08-31T15:03:04Z</time>
    </trkpt>
    <trkpt lat="43.0239081" lon="-88.0535405">
      <time>2017-08-31T15:14:28Z</time>
    </trkpt>
    <trkpt lat="43.0156329" lon="-88.0469584">
      <time>2017-08-31T15:18:00Z</time>
    </trkpt>
    <trkpt lat="43.0083531" lon="-88.0556531">
      <time>2017-08-31T15:23:00Z</time>
    </trkpt>
    <trkpt lat="43.0075324" lon="-88.0555673">
      <time>2017-08-31T15:24:46Z</time>
    </trkpt>
    <trkpt lat="43.008418" lon="-88.0557401">
      <time>2017-08-31T15:31:52Z</time>
    </trkpt>
    <trkpt lat="43.0126481" lon="-88.0568062">
      <time>2017-08-31T15:33:00Z</time>
    </trkpt>
    <trkpt lat="43.0128916" lon="-88.0518452">
      <time>2017-08-31T15:38:00Z</time>
    </trkpt>
    <trkpt lat="43.0127273" lon="-88.0521107">
      <time>2017-08-31T15:38:40Z</time>
    </trkpt>
    <trkpt lat="43.0131722" lon="-88.0527378">
      <time>2017-08-31T15:51:02Z</time>
    </trkpt>
    <trkpt lat="43.0134494" lon="-88.0470176">
      <time>2017-08-31T15:53:00Z</time>
    </trkpt>
    <trkpt lat="43.0025737" lon="-88.0317175">
      <time>2017-08-31T15:58:00Z</time>
    </trkpt>
    <trkpt lat="43.0029008" lon="-87.9864016">
      <time>2017-08-31T16:03:00Z</time>
    </trkpt>
    <trkpt lat="43.0029048" lon="-87.9856561">
      <time>2017-08-31T16:03:04Z</time>
    </trkpt>
    <trkpt lat="43.0078832" lon="-87.9755603">
      <time>2017-08-31T16:07:20Z</time>
    </trkpt>
    <trkpt lat="43.0082468" lon="-87.9746562">
      <time>2017-08-31T16:45:14Z</time>
    </trkpt>
    <trkpt lat="43.0101215" lon="-87.9678956">
      <time>2017-08-31T16:48:00Z</time>
    </trkpt>
    <trkpt lat="43.010196" lon="-87.9617795">
      <time>2017-08-31T16:48:54Z</time>
    </trkpt>
    <trkpt lat="43.010231" lon="-87.9607323">
      <time>2017-08-31T17:00:12Z</time>
    </trkpt>
    <trkpt lat="43.0029334" lon="-87.9659848">
      <time>2017-08-31T17:03:00Z</time>
    </trkpt>
    <trkpt lat="43.0029545" lon="-87.9666883">
      <time>2017-08-31T17:03:04Z</time>
    </trkpt>
    <trkpt lat="43.0029545" lon="-87.9666883">
      <time>2017-08-31T17:03:06Z</time>
    </trkpt>
    <trkpt lat="43.0029532" lon="-87.967987">
      <time>2017-08-31T17:03:38Z</time>
    </trkpt>
    <trkpt lat="43.0028567" lon="-87.9956577">
      <time>2017-08-31T17:08:00Z</time>
    </trkpt>
    <trkpt lat="43.0026913" lon="-88.0268902">
      <time>2017-08-31T17:13:00Z</time>
    </trkpt>
    <trkpt lat="43.0027095" lon="-88.0523474">
      <time>2017-08-31T17:18:00Z</time>
    </trkpt>
    <trkpt lat="43.004406" lon="-88.054555">
      <time>2017-08-31T17:19:46Z</time>
    </trkpt>
    <trkpt lat="43.0052889" lon="-88.0546366">
      <time>2017-08-31T17:39:02Z</time>
    </trkpt>
    <trkpt lat="43.0025914" lon="-88.0319">
      <time>2017-08-31T17:43:00Z</time>
    </trkpt>
    <trkpt lat="43.0115608" lon="-88.0079276">
      <time>2017-08-31T17:48:00Z</time>
    </trkpt>
    <trkpt lat="43.0191452" lon="-87.9782634">
      <time>2017-08-31T17:53:00Z</time>
    </trkpt>
    <trkpt lat="43.0217911" lon="-87.9539831">
      <time>2017-08-31T17:58:00Z</time>
    </trkpt>
    <trkpt lat="43.0231886" lon="-87.9196989">
      <time>2017-08-31T18:03:00Z</time>
    </trkpt>
    <trkpt lat="43.0232477" lon="-87.9184276">
      <time>2017-08-31T18:03:18Z</time>
    </trkpt>
    <trkpt lat="43.0232854" lon="-87.9183973">
      <time>2017-08-31T18:03:36Z</time>
    </trkpt>
    <trkpt lat="43.0232475" lon="-87.9173488">
      <time>2017-08-31T18:03:56Z</time>
    </trkpt>
    <trkpt lat="43.0262895" lon="-87.9121198">
      <time>2017-08-31T18:08:00Z</time>
    </trkpt>
    <trkpt lat="43.0261065" lon="-87.9118902">
      <time>2017-08-31T18:10:14Z</time>
    </trkpt>
    <trkpt lat="43.0267069" lon="-87.9125916">
      <time>2017-08-31T18:18:24Z</time>
    </trkpt>
    <trkpt lat="43.0320941" lon="-87.9094621">
      <time>2017-08-31T18:23:00Z</time>
    </trkpt>
    <trkpt lat="43.0288772" lon="-87.9075285">
      <time>2017-08-31T18:28:00Z</time>
    </trkpt>
    <trkpt lat="43.0251038" lon="-87.9052248">
      <time>2017-08-31T18:30:02Z</time>
    </trkpt>
    <trkpt lat="43.0251932" lon="-87.9062828">
      <time>2017-08-31T18:41:18Z</time>
    </trkpt>
    <trkpt lat="43.0251756" lon="-87.9062244">
      <time>2017-08-31T18:43:12Z</time>
    </trkpt>
    <trkpt lat="43.0251311" lon="-87.9051873">
      <time>2017-08-31T18:57:12Z</time>
    </trkpt>
    <trkpt lat="43.0263937" lon="-87.9053374">
      <time>2017-08-31T18:58:00Z</time>
    </trkpt>
    <trkpt lat="43.0365599" lon="-87.9031771">
      <time>2017-08-31T19:02:34Z</time>
    </trkpt>
    <trkpt lat="43.0365606" lon="-87.9031306">
      <time>2017-08-31T19:03:06Z</time>
    </trkpt>
    <trkpt lat="43.0364767" lon="-87.9042894">
      <time>2017-08-31T19:03:32Z</time>
    </trkpt>
    <trkpt lat="43.0062183" lon="-87.8918648">
      <time>2017-08-31T19:08:00Z</time>
    </trkpt>
    <trkpt lat="42.9592299" lon="-87.8846378">
      <time>2017-08-31T19:13:00Z</time>
    </trkpt>
    <trkpt lat="42.9586454" lon="-87.9163934">
      <time>2017-08-31T19:18:00Z</time>
    </trkpt>
    <trkpt lat="42.9562396" lon="-87.9161331">
      <time>2017-08-31T19:22:50Z</time>
    </trkpt>
    <trkpt lat="42.9569091" lon="-87.9155129">
      <time>2017-08-31T19:35:56Z</time>
    </trkpt>
    <trkpt lat="42.9584507" lon="-87.9154828">
      <time>2017-08-31T19:36:26Z</time>
    </trkpt>
    <trkpt lat="42.9592366" lon="-87.9155177">
      <time>2017-08-31T19:43:56Z</time>
    </trkpt>
    <trkpt lat="42.9593697" lon="-87.9299871">
      <time>2017-08-31T19:45:40Z</time>
    </trkpt>
    <trkpt lat="42.97543" lon="-87.9154267">
      <time>2017-08-31T19:48:00Z</time>
    </trkpt>
    <trkpt lat="43.0360168" lon="-87.928499">
      <time>2017-08-31T19:53:00Z</time>
    </trkpt>
    <trkpt lat="43.0348143" lon="-87.9343348">
      <time>2017-08-31T19:57:00Z</time>
    </trkpt>
    <trkpt lat="43.0348728" lon="-87.9343475">
      <time>2017-08-31T20:03:04Z</time>
    </trkpt>
    <trkpt lat="43.0347717" lon="-87.9353638">
      <time>2017-08-31T20:20:14Z</time>
    </trkpt>
    <trkpt lat="43.0352277" lon="-87.9264918">
      <time>2017-08-31T20:23:00Z</time>
    </trkpt>
    <trkpt lat="43.0599605" lon="-87.9379952">
      <time>2017-08-31T20:28:00Z</time>
    </trkpt>
    <trkpt lat="43.0761479" lon="-87.958543">
      <time>2017-08-31T20:33:00Z</time>
    </trkpt>
    <trkpt lat="43.0953096" lon="-87.983033">
      <time>2017-08-31T20:38:00Z</time>
    </trkpt>
    <trkpt lat="43.1256024" lon="-88.0058333">
      <time>2017-08-31T20:43:00Z</time>
    </trkpt>
    <trkpt lat="43.1484166" lon="-88.0052597">
      <time>2017-08-31T20:48:00Z</time>
    </trkpt>
    <trkpt lat="43.1496855" lon="-88.0157592">
      <time>2017-08-31T20:51:24Z</time>
    </trkpt>
    <trkpt lat="43.1496387" lon="-88.0157251">
      <time>2017-08-31T21:03:04Z</time>
    </trkpt>
    <trkpt lat="43.1489665" lon="-88.0163915">
      <time>2017-08-31T21:04:02Z</time>
    </trkpt>
    <trkpt lat="43.1351389" lon="-88.0057463">
      <time>2017-08-31T21:08:00Z</time>
    </trkpt>
    <trkpt lat="43.1321613" lon="-87.9987111">
      <time>2017-08-31T21:11:34Z</time>
    </trkpt>
    <trkpt lat="43.1331105" lon="-87.9985976">
      <time>2017-08-31T21:26:16Z</time>
    </trkpt>
    <trkpt lat="43.1342334" lon="-88.0039219">
      <time>2017-08-31T21:28:00Z</time>
    </trkpt>
    <trkpt lat="43.156088" lon="-88.0062576">
      <time>2017-08-31T21:33:00Z</time>
    </trkpt>
    <trkpt lat="43.1537617" lon="-88.0056933">
      <time>2017-08-31T21:40:58Z</time>
    </trkpt>
    <trkpt lat="43.1543852" lon="-88.0055853">
      <time>2017-08-31T21:49:20Z</time>
    </trkpt>
    <trkpt lat="43.1568837" lon="-88.0122886">
      <time>2017-08-31T21:53:00Z</time>
    </trkpt>
    <trkpt lat="43.1569529" lon="-88.0125847">
      <time>2017-08-31T21:53:50Z</time>
    </trkpt>
    <trkpt lat="43.1570219" lon="-88.012591">
      <time>2017-08-31T22:03:04Z</time>
    </trkpt>
    <trkpt lat="43.1568351" lon="-88.0115547">
      <time>2017-08-31T22:12:28Z</time>
    </trkpt>
    <trkpt lat="43.1484302" lon="-87.9827013">
      <time>2017-08-31T22:18:00Z</time>
    </trkpt>
    <trkpt lat="43.1371659" lon="-87.9538692">
      <time>2017-08-31T22:23:00Z</time>
    </trkpt>
    <trkpt lat="43.1021863" lon="-87.9454563">
      <time>2017-08-31T22:28:00Z</time>
    </trkpt>
    <trkpt lat="43.087122" lon="-87.9308695">
      <time>2017-08-31T22:33:00Z</time>
    </trkpt>
    <trkpt lat="43.0698662" lon="-87.920871">
      <time>2017-08-31T22:38:00Z</time>
    </trkpt>
    <trkpt lat="43.0411356" lon="-87.9251849">
      <time>2017-08-31T22:43:00Z</time>
    </trkpt>
    <trkpt lat="43.0100252" lon="-87.9162395">
      <time>2017-08-31T22:48:00Z</time>
    </trkpt>
    <trkpt lat="42.9823403" lon="-87.9137087">
      <time>2017-08-31T22:53:00Z</time>
    </trkpt>
    <trkpt lat="42.9737793" lon="-87.9288849">
      <time>2017-08-31T22:58:00Z</time>
    </trkpt>
    <trkpt lat="42.9591459" lon="-87.9281101">
      <time>2017-08-31T23:01:16Z</time>
    </trkpt>
    <trkpt lat="42.9591356" lon="-87.9281184">
      <time>2017-08-31T23:03:02Z</time>
    </trkpt>
    <trkpt lat="42.9592854" lon="-87.9266011">
      <time>2017-08-31T23:38:24Z</time>
    </trkpt>
    <trkpt lat="42.9582797" lon="-87.9167001">
      <time>2017-08-31T23:43:00Z</time>
    </trkpt>
    <trkpt lat="42.9583198" lon="-87.9169082">
      <time>2017-08-31T23:50:42Z</time>
    </trkpt>
    <trkpt lat="42.9583411" lon="-87.9169085">
      <time>2017-08-31T23:51:48Z</time>
    </trkpt>
  </trkseg>
</trk>
</gpx>
```

and run it against

https://maps.openrouteservice.org/directions?n1=43.004898&n2=-87.82093&n3=11&a=42.958337,-87.916885,42.959563,-87.948623,43.033379,-87.95775,43.035094,-88.036668,43.012648,-88.056806,43.002574,-88.031717,43.010231,-87.960732,43.002933,-87.965985,43.004406,-88.054555,43.023248,-87.917349,43.032094,-87.909462,43.025104,-87.905225,43.036561,-87.903131,42.95923,-87.884638,42.95937,-87.929987,42.97543,-87.915427,43.034772,-87.935364,43.035228,-87.926492,43.059961,-87.937995,43.125602,-88.005833,43.148417,-88.00526,43.149686,-88.015759,43.133111,-87.998598,43.157022,-88.012591,43.137166,-87.953869,43.102186,-87.945456,43.069866,-87.920871,42.98234,-87.913709,42.973779,-87.928885,42.959136,-87.928118,42.958341,-87.916908&b=0&c=0&k1=en-US&k2=km

#### Here's what I think could be improved

Answer given by Adam :

it seems the out of bounds thing is a bug in the code in openrouteservice that generates the GPX as it works with json and geojson output format (I assume it is an issue relating to the code using waypoints and not including them correctly when it comes to formatting the output for GPX).

It would be great if you could create a bug issue on our GitHub page so that one of the developers can look into it (https://github.com/GIScience/openrouteservice).

But the fact that you got the out of bounds error means that it got passed the checks for estimated distance and so your config is fine for that (I believe it would be the maximum_distance value that controls the distance limit here).

**Comment by takb** [comment]
I tried around a bit to reproduce this but couldn't. Error code 2004 does list the actual limit the backend compares the distance approximation against, which means your parameter `ors.services.routing.profiles.default_params.maximum_distance = 200000` is overridden somewhere else, most likely in the respective profile's settings that you are querying against. 
I verified that with our current code version the maximum_distance value is handled correctly for the directions endpoint regardless which output format is specified. 

About the 'Java error about being out of boundaries', I'd need to see the full actual error message to investigate further.
